pkg/project/api/install/install.go

```diff
@@ -33,7 +33,7 @@ func init() {
 		}
 	}
 	if len(externalVersions) == 0 {
-		glog.Infof("No version is registered for group %v", api.GroupName)
+		glog.Infof("No version is registered for group %q", api.LegacyGroupName)
```

Comment:
well, %v in case of empty string will be empty string, so that will make this error message look pretty weird.